src/main/java/fr/insalyon/creatis/moteurlite/MoteurLite.java

```diff
@@ -0,0 +1,76 @@
+package fr.insalyon.creatis.moteurlite;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import fr.insalyon.creatis.gasw.Gasw;
+import fr.insalyon.creatis.moteurlite.boutiquesParser.ParseBoutiquesFile;
+import fr.insalyon.creatis.moteurlite.inputsParser.ParseInputsFile;
+import fr.insalyon.creatis.moteurlite.iterationStrategy.IterationStrategy;
+
+/**
+ * 
+ * @author Sandesh Patil [https://github.com/sandepat]
+ * 
+ */
+
+public class MoteurLite {
+    private Gasw gasw = Gasw.getInstance();
+    private int sizeOfInputs;
+    private String bashScript;
+    IterationStrategy iterationStrategy = new IterationStrategy();
+    Workflowsdb workflowsdb = new Workflowsdb();
+
+    public static void main(String[] args) throws Exception {
+        new MoteurLite(args);
+    }
+
+    public MoteurLite(String[] args) throws Exception {
+        String workflowId = args[0];
+        String boutiquesFilePath = args[1];
+        String inputsFilePath = args[2];
```

Comment:
Added a condition to check if there are exactly three arguments, otherwise throw IllegalArgumentException

Added a condition to check if any of the arguments are empty, otherwise IllegalArgumentException.

Added a method isValidWorkflowId, which checks whether the workflowId is valid by ensuring it is neither null nor an empty string (after trimming any leading or trailing whitespace).

Added Validation for boutiquesFilePath: Ensures it ends with .json.

Added Validation for inputsFilePath: Ensures it ends with .xml.